nodeadm/test/e2e/cases/kubelet-flags/config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: node.eks.aws/v1alpha1
+kind: NodeConfig
+spec:
+  cluster:
+    name: my-cluster
+    apiServerEndpoint: https://example.com
+    certificateAuthority: Y2VydGlmaWNhdGVBdXRob3JpdHk=
+    cidr: 10.100.0.0/16
```

Comment:
Does this cause NodeConfig to inject kubelet flags? What if they conflict with user defined flags?

Reply:
[carter just answered this one](https://github.com/awslabs/amazon-eks-ami/pull/1593#discussion_r1467083198) while we determined the input style; we will generate flags in nodeadm and then user-provided flags can override any of these